charts/timescaledb-single/admin-guide.md

```diff
@@ -182,22 +184,29 @@ This allows you to decouple the secret management of the backup credentials from
 The [pgBackRest Command Reference](https://pgbackrest.org/command.html#introduction) has detailed information about which
 environment variables can be set.
 
-For example, if you would use [Vault](https://www.vaultproject.io/), you could do the following:
```

Comment:
I fully removed Vault in favour of using plain secrets, as plain secrets are available to everyone, yet Vault is very specific.